### PR TITLE
minor fixes to splines example

### PR DESCRIPTION
**Pull request by drbenvincent**
ref #234 

Minor fixes:
- Change PyMC3 to PyMC in the notebook title
- Remove `pm.*` from tags

### CODE REVIEW

File: examples/case_studies/spline.ipynb [hunk 1]
```diff
@@ -4,12 +4,12 @@
    "cell_type": "markdown",
```
Comment (twiecki) [inline examples/case_studies/spline.ipynb:4]: <p>Can we find a better title that will help also in the gallery as the title? Maybe just "Splines"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (drbenvincent) [reply examples/case_studies/spline.ipynb:4]: Have updated front matter to be more correct. Also changed title to just "Splines"

File: myst_nbs/case_studies/spline.myst.md [hunk 2]
```diff
@@ -11,12 +11,13 @@ kernelspec:
...
+(spline)=
+# Splines
 
-:::{post} May 6, 2022 
-:tags: patsy, pymc.Deterministic, pymc.Exponential, pymc.Model, pymc.Normal, regression, spline
+:::{post} June 4, 2022 
+:tags: patsy, regression, spline
 :category: beginner
-:author: Joshua Cook updated by Tyler James Burch, Chris Fonnesbeck
+:author: Joshua Cook, Tyler James Burch, Chris Fonnesbeck, Benjamin Vincent
```
Comment (OriolAbril) [inline myst_nbs/case_studies/spline.myst.md:20]: I am completely unaware of how/if we are enforcing this, especially during this last sprint week. I generally try to comment on that to make sure people who should but haven't yet add themselves and viceversa. According to what we decided and wrote in: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#authorship-and-attribution and last sentence in https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell. You should not add yourself to the metadata, we don't have a word defined for fixing typos and such, we could add one and add yourself to the authorship section

Comment (drbenvincent) [reply myst_nbs/case_studies/spline.myst.md:20]: Want quite sure about the meta data versus the authorship at the end.  Totally happy to remove myself.

**Review by aloctavodia — APPROVED** [review]

**Pull request merged**